package.json

```diff
@@ -23,7 +23,7 @@
   "dependencies": {
     "JSON2": "~0.1.0",
     "applicationinsights": "~0.17.0",
-    "async": "^2.1.4",
+    "async": "~2.1.4",
```

Comment:
Do you think it's safe to change it minor version update? I've seen semver misused and things getting broken because of silent update. I wonder if minor updates should be manual.